servicecertsigner/v1alpha1/types.go

```diff
@@ -89,13 +70,24 @@ type ServiceCertSignerOperatorConfigSpec struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	ServiceServingCertSignerConfig runtime.RawExtension `json:"serviceServingCertSignerConfig"`
 
+	// caBundleInjectorConfig holds a sparse config that the user wants for this component.  It only needs to be the overrides from the defaults
+	// it will end up overlaying in the following order:
+	// 1. hardcoded default
+	// 2. this config
+	// +kubebuilder:pruning:PreserveUnknownFields
+	CABundleInjectorConfig runtime.RawExtension `json:"caBundleInjectorConfig"`
+
+	// apiServiceCABundleInjectorConfig has been deprecated in favor of caBundleInjectorConfig.
+	//
 	// apiServiceCABundleInjectorConfig holds a sparse config that the user wants for this component.  It only needs to be the overrides from the defaults
 	// it will end up overlaying in the following order:
 	// 1. hardcoded default
 	// 2. this config
 	// +kubebuilder:pruning:PreserveUnknownFields
 	APIServiceCABundleInjectorConfig runtime.RawExtension `json:"apiServiceCABundleInjectorConfig"`
 
+	// configMapCABundleInjectorConfig has been deprecated in favor of caBundleInjectorConfig.
```

Comment:
isn't there some Deprecated syntax we usually use in APIs?

Reply:
`DEPRECATED[: <reason]` appears to be the most common form.